src/ExternalApi/Isite/Mapper/ContentBlockMapper.php

```diff
@@ -257,7 +257,7 @@ public function getDomainModel(SimpleXMLElement $isiteObject)
             case 'promotions':
                 $contentBlockData = $form->content;
                 $title = $this->getString($contentBlockData->title);
-                $layout = $this->getString($contentBlockData->layout);
+                $layout = $this->getString($contentBlockData->layout) ?: 'list'; // Default to list if unset
```

Comment:
This looks kinda weird. Does this work in PHP, I know it would in JS:
```suggestion
                $layout = $this->getString($contentBlockData->layout) || 'list'; // Default to list if unset
```